include/aligator/solvers/proxddp/solver-proxddp.hpp

```diff
@@ -227,12 +227,10 @@ template <typename _Scalar> struct SolverProxDDPTpl {
   /// \{
 
   /// @brief    Add a callback to the solver instance.
-  void registerCallback(const std::string &name, CallbackPtr cb) {
-    callbacks_[name] = cb;
-  }
+  void registerCallback(const std::string &name, CallbackPtr cb);
 
   /// @brief    Remove all callbacks from the instance.
-  void clearCallbacks() noexcept { callbacks_.clear(); }
+  inline void clearCallbacks() noexcept { callbacks_.clear(); }
```

Comment:
Is it relevant to add the inline keyword to other functions in order to reduce the number of function calls and speed up execution?

Reply:
Actually, it's not useful here at all since it's all template code. Modern compilers would inline this anyway
```suggestion
  void clearCallbacks() noexcept { callbacks_.clear(); }
```